frontend/TestRun/StructuredEvent.svelte

```diff
@@ -29,11 +29,9 @@
     <div class="event-header d-flex align-items-start flex-wrap">
         <div class="ms-2 mb-2 bg-dark text-light rounded px-2">{event.eventType}</div>
         <div class="ms-2 mb-2 rounded px-2 severity-{event.severity.toLowerCase()}">{event.severity}</div>
-        {#if event.nemesis}
-            <div class="ms-2 mb-2 rounded px-2 status-{event.nemesis.status.toLowerCase()}">{event.nemesis.name}</div>
-        {:else}
-            <div class="ms-2 mb-2 rounded px-2 status-skipped">no nemesis</div>
-        {/if}
+        {#each event.nemesis ?? [] as nemesis}
+            <div class="ms-2 mb-2 rounded px-2 status-{nemesis.status.toLowerCase()}">{nemesis.name}</div>
```

Comment:
we lose the note about no nemesis

Reply:
Do you think we need to have that? If there's no note I feel it would serve the same purpose.

Reply:
I don't mind, just highlighting it. We can try without it.

Reply:
I don't think anyone was really reading it, and would expect it